### PR TITLE
[PFS-237] Add Direct Subvenance to CommitInfo Protos

### PR DESCRIPTION
**Pull request by FahadBSyed**

### CODE REVIEW

**Review by emmajsadams — APPROVED** [review]

File: src/pfs/pfs.proto [hunk 1]
```diff
@@ -197,6 +197,8 @@ message CommitInfo {
   google.protobuf.Timestamp finishing = 7;
   google.protobuf.Timestamp finished = 8;
   repeated Commit direct_provenance = 13;
+  repeated Commit direct_subvenance = 15;
```
Comment (bbonenfant) [inline src/pfs/pfs.proto:200]: did you run the python proto generation? I would expect this change to result in a diff within `python-sdk/api/pfs/__init__.py`

Comment (FahadBSyed) [reply src/pfs/pfs.proto:200]: Sorry I couldn't get the make proto target to work anymore so I generated the protos locally. Do we know how to run the python proto generation?

Comment (FahadBSyed) [reply src/pfs/pfs.proto:200]: Okay, I fixed the generation on my end, the files should be there now

**Review by bbonenfant — APPROVED** [review]

File: src/internal/pfsdb/commits.go [hunk 2]
```diff
@@ -726,6 +726,15 @@ func getCommitInfoFromCommitRow(ctx context.Context, extCtx sqlx.ExtContext, row
 	if err != nil {
 		return nil, errors.Wrap(err, "get provenance for commit")
 	}
+	subvenantCommits, err := getCommitSubvenance(ctx, extCtx, row.ID)
```
Comment (brycemcanally) [inline src/internal/pfsdb/commits.go:729]: I think we are going to need to set the depth option to 1 here.

Comment (brycemcanally) [reply src/internal/pfsdb/commits.go:729]: Adding some tests for the direct provenance / subvenance would be helpful.

Comment (FahadBSyed) [reply src/internal/pfsdb/commits.go:729]: Yup, you're right. I'm not sure why that slipped my mind.

**Pull request merged**